.github/workflows/release.yml

```diff
@@ -65,6 +65,7 @@ jobs:
         run: |
           cd osxcross
           wget https://github.com/phracker/MacOSX-SDKs/releases/download/11.3/MacOSX11.3.sdk.tar.xz -O tarballs/MacOSX11.3.sdk.tar.xz
+          echo cd4f08a75577145b8f05245a2975f7c81401d75e9535dcffbb879ee1deefcbf4 tarballs/MacOSX11.3.sdk.tar.xz | sha256sum -c -
```

Comment:
could we make this a constant instead of inline?

Reply:
done